localisation/00_memewc_united_localisation.csv

```diff
@@ -173,24 +173,24 @@ mace_onehanded_4_desc;Epic One-handed Mace.;;Epischer Streitkolben.;;;;;;;;;;;x
-halberd_5_desc;Legendary Halberd.;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-spear_1;[GetArtifactWeaponsCommonPrefix] Spear;;;;;;;;;;;;;x
-spear_1_desc;Common Spear.;;;;;;;;;;;;;x
-spear_2;[GetArtifactWeaponsUncommonPrefix][GetArtifactWeaponsUncommonSuffix] Spear;;;;;;;;;;;;;x
-spear_2_desc;Uncommon Spear.;;;;;;;;;;;;;x
-spear_3;[GetArtifactWeaponsRarePrefix][GetArtifactWeaponsRareSuffix] Spear;;;;;;;;;;;;;x
-spear_3_desc;Rare Spear.;;;;;;;;;;;;;x
-spear_4;[GetArtifactWeaponsEpicPrefix][GetArtifactWeaponsEpicSuffix] Spear;;;;;;;;;;;;;x
-spear_4_desc;Epic Spear.;;;;;;;;;;;;;x
-spear_5;[GetArtifactWeaponsLegendaryPrefix][GetArtifactWeaponsLegendarySuffix] Spear;;;;;;;;;;;;;x
-spear_5_desc;Legendary Spear.;;;;;;;;;;;;;x
\ No newline at end of file
+halberd_1;[GetArtifactWeaponsCommonPrefix] Halberd;;[GetArtifactWeaponsCommonPrefix]e Hellebarde;;;;;;;;;;;x
+halberd_1_desc;Common Halberd.;;Gew�hnliche Hellebarde.;;;;;;;;;;;x
+halberd_2;[GetArtifactWeaponsUncommonPrefix][GetArtifactWeaponsUncommonSuffix] Halberd;;[GetArtifactWeaponsUncommonPrefix]e Hellebarde[GetArtifactWeaponsUncommonSuffix];;;;;;;;;;;x
+halberd_2_desc;Uncommon Halberd.;;Ungew�hnliche Hellebarde.;;;;;;;;;;;x
+halberd_3;[GetArtifactWeaponsRarePrefix][GetArtifactWeaponsRareSuffix] Halberd;;[GetArtifactWeaponsRarePrefix]e Hellebarde[GetArtifactWeaponsRareSuffix];;;;;;;;;;;x
+halberd_3_desc;Rare Halberd.;;Seltene Hellebarde.;;;;;;;;;;;x
+halberd_4;[GetArtifactWeaponsEpicPrefix][GetArtifactWeaponsEpicSuffix] Halberd;;[GetArtifactWeaponsEpicPrefix]e Hellebarde[GetArtifactWeaponsEpicSuffix];;;;;;;;;;;x
+halberd_4_desc;Epic Halberd.;;Epische Hellebarde.;;;;;;;;;;;x
+halberd_5;[GetArtifactWeaponsLegendaryPrefix][GetArtifactWeaponsLegendarySuffix] Halberd;;[GetArtifactWeaponsLegendaryPrefix]e Hellebarde[GetArtifactWeaponsLegendarySuffix];;;;;;;;;;;x
+halberd_5_desc;Legendary Halberd.;;Legend�re Hellebarde.;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+spear_1;[GetArtifactWeaponsCommonPrefix] Spear;;[GetArtifactWeaponsCommonPrefix]er Speer;;;;;;;;;;;x
+spear_1_desc;Common Spear.;;Gew�hnlicher Speer.;;;;;;;;;;;x
+spear_2;[GetArtifactWeaponsUncommonPrefix][GetArtifactWeaponsUncommonSuffix] Spear;;[GetArtifactWeaponsUncommonPrefix]er Speer[GetArtifactWeaponsUncommonSuffix];;;;;;;;;;;x
+spear_2_desc;Uncommon Spear.;;Ungew�hnlicher Speer.;;;;;;;;;;;x
+spear_3;[GetArtifactWeaponsRarePrefix][GetArtifactWeaponsRareSuffix] Spear;;[GetArtifactWeaponsRarePrefix]er Speer[GetArtifactWeaponsRareSuffix];;;;;;;;;;;x
+spear_3_desc;Rare Spear.;;Seltener Speer.;;;;;;;;;;;x
+spear_4;[GetArtifactWeaponsEpicPrefix][GetArtifactWeaponsEpicSuffix] Spear;;[GetArtifactWeaponsEpicPrefix]er Speer[GetArtifactWeaponsEpicSuffix];;;;;;;;;;;x
+spear_4_desc;Epic Spear.;;Epischer Speer.;;;;;;;;;;;x
+spear_5;[GetArtifactWeaponsLegendaryPrefix][GetArtifactWeaponsLegendarySuffix] Spear;;[GetArtifactWeaponsLegendaryPrefix]er Speer[GetArtifactWeaponsLegendarySuffix];;;;;;;;;;;x
+spear_5_desc;Legendary Spear.;;Legend�rer Speer.;;;;;;;;;;;x
```

Comment:
Are you sure that this is correct?
If anything, `GetArtifactWeapons%%Suffix` here is second prefix, so `[GetArtifactWeaponsUncommonPrefix][GetArtifactWeaponsUncommonSuffix] Halberd`, for example, can be `Barbaric Ritual Halberd`.

Reply:
Yes, because I used in german phrases like "of strength" as suffix, which come after the weapon type. I know, it's different from your english original, but makes more sense in german. And is used like that in a lot of RPG weapon namings...

Reply:
Tested ingame, works as intended. Just have to write descriptions for it yet...

Reply:
> it's different from your english original

Nah, that's fine as long as it sounds good in german